### DISCUSSION TITLE
Fourth Community Meeting 28/7/2022

### DISCUSSION THREAD

**Discussion by VojtechBartoska** [espressif/arduino-esp32#6927] [General]
Hello to all Arduino ESP32 Core Users 🚀 

**Meeting purpose,**
we appreciate all of your contribution and help and we would like to improve your User experience on this Project. 🤩

As a part of this improvement we would like to uncover our internal Development to you, have clear and transparent planning and also listen to your requests, ideas and valuable Feedback.

To fulfill this, we would like to held every Month Community Meetings where we can share main updates and also have an open discussion with you.

---

**💥Fourth Community Meeting will take place on Thursday 28/07/2022 at 4 pm GMT+2 time zone. (1 hour length) We will use Google Meet.**

---

We want to have some idea about how many of you would be interested in joining these Meetings so instead of sharing directly link to Online Meeting itself, it will work by invitation.

**If you want to be added to upcoming Meeting and receive invitation please send an email to [vojtech.bartoska@espressif.com](mailto:vojtech.bartoska@espressif.com).** 

If you participated on any Community Meeting, you are automatically invited to upcoming Meeting and it's up to you if you joins us again or not.

---

### 1) Espressif Introduction (3 minutes) 
Meeting purpose & rules
### 2) Development updates (5 minutes)
Summary of our current development and news
### 2) Community topics
Up to your proposals in the comments.
### 3) Open Discussions
Place for any topic which is not in Agenda.

See you on the Meeting!

**Comment by torntrousers** [comment]
Appologies, I cannot make it today.

**Reply by VojtechBartoska** [reply]
Don't worry, see you next time!

**Reply by VojtechBartoska** [reply]
Also this was supposed to be posted in Third Meeting topic [here](https://github.com/espressif/arduino-esp32/discussions/6811).

Just explaining it so it's not bringing confusion and this thread is for upcoming Meeting and it's still actual. :)

**Comment by VojtechBartoska** [comment]
Slides: 
[Fourth_Community_meeting-Arduino-July_2022.pdf](https://github.com/espressif/arduino-esp32/files/9679117/Fourth_Community_meeting-Arduino-July_2022.pdf)
